webSocket/server/transceivers_test.go

```diff
@@ -0,0 +1,33 @@
+package server
+
+import (
+	"testing"
+
+	"github.com/multiversx/mx-chain-core-go/testscommon"
+	"github.com/stretchr/testify/require"
+)
+
+func TestReceiversHolderAddAndRemove(t *testing.T) {
+	t.Parallel()
+
+	recsHolder := NewTransceiversAndConnHolder()
+
+	recsHolder.AddTransceiverAndConn(&testscommon.WebSocketReceiverStub{}, &testscommon.WebsocketConnectionStub{
+		GetIDCalled: func() string {
+			return "id1"
+		},
+	})
+	recsHolder.AddTransceiverAndConn(&testscommon.WebSocketReceiverStub{}, &testscommon.WebsocketConnectionStub{
+		GetIDCalled: func() string {
+			return "id2"
+		},
+	})
+
+	recsHolder.Remove("id1")
```

Comment:
missing test here when you have 2 transceivers

Reply:
added